scale/ingest/test/test_models.py

```diff
@@ -78,24 +77,6 @@ def setUp(self):
         self.workspace = storage_test_utils.create_workspace()
         self.recipe = recipe_test_utils.create_recipe_type_v6()
 
```

Comment:
Are we removing support for all schemas that are not v6? If so, we need to make sure we can and do migrate all existing configs, definitions, etc. stored in the database and elsewhere to v6.

Reply:
To be handled in #1551